lib/base-frames.ts

```diff
@@ -64,6 +67,44 @@ export class AMQPMethodFrame {
     }
 }
 
+export class AMQPMethodFrame extends AMQPFrame {
+
+    constructor(
+        private readonly _classId: AMQPClassesId,
+        private readonly _methodId: AMQPChannelMethod | AMQPConnectionMethod | AMQPBasicMethod,
+        private readonly _channel: number
+    ){
+        super(AMQPFrameType.METHOD, _channel);
+        this._buffer.writeUInt16BE(_classId, this._currentOffset); this._currentOffset += TWO_OCTET;
+        this._buffer.writeUInt16BE(_methodId, this._currentOffset); this._currentOffset += TWO_OCTET;
+    }
+}
+
+export class AMQPContentHeaderFrame<T extends Record<string, AMQPDataType>> extends AMQPFrame {
+    constructor(private _channel: number, payloadSize: number, propertyFlag: number, headers: Table<T>){
+        super(AMQPFrameType.CONTENT_HEADER, _channel);
+        this._buffer.writeUInt16BE(AMQPClassesId.BASIC, this._currentOffset); this._currentOffset += TWO_OCTET;
+        this.apply(new LongInt(0));
+        this.apply(new Long64Int(payloadSize));
+        this.apply(new ShortInt(2000));
```

Comment:
[For reference](https://github.com/cloudamqp/amqp-client.js/blob/main/src/amqp-view.ts#L159)

We may need some Bits operations here